en/appendices/3-5-migration-guide.rst

```diff
@@ -180,6 +180,7 @@ Routing
   match.
 * ``Cake\Routing\RouteBuilder`` now has methods to create routes for
   specific HTTP methods. e.g ``get()`` and ``post()``.
+* ``Cake\Routing\RouteBuilder::loadPlugin()`` was added.
```

Comment:
I'm thinking we don't deprecate it based on the discussion we had about making plugin routes more complicated.